tests/e2e/distributed_queries/workloads.yaml

```diff
@@ -11,3 +11,23 @@ workloads:
     binary: "tests/e2e/pytest_runner.sh"
     args: ["distributed_queries/distributed_queries.py"]
     <<: *template_cluster
+
+  - name: "Distributed unwind collect"
```

Comment:
I is possible to move all of these tests into the original `distributed_queries.py` file? I feel that creating different workloads for such small tests are unnecessary. I know creating multiple tests in the same file is not viable (because we cannot delete the vertices/edges), but I think these are tests that will be removed in the long term as the cypher test cases cover them much better so it is okay to have a single, maybe a bit longer test case to check all of these.

Reply:
For now I rather not as we don't have `detatch delete`. 
> but I think these are tests that will be removed in the long term
I will do it myself when this is the case.